starter-core/src/test/groovy/io/micronaut/starter/feature/micrometer/MicrometerSpec.groovy

```diff
@@ -29,6 +29,19 @@ class MicrometerSpec extends ApplicationContextSpec {
         micrometerFeature << beanContext.getBeansOfType(MicrometerFeature).iterator()
     }
 
+    void 'test gradle micrometer feature oracle cloud'() {
+        given:
+        OracleCloud oracleCloudFeature = beanContext.getBean(OracleCloud)
+
+        when:
+        String template = new BuildBuilder(beanContext, BuildTool.GRADLE)
+                .features([oracleCloudFeature.name])
+                .render()
+
+        then:
+        template.contains('implementation("io.micronaut.oraclecloud:micronaut-oraclecloud-bmc-monitoring")')
```

Comment:
Please, use `BuildTestVerifier` instead to test dependency in every possible build tool.